doc/examples/applications/plot_rank_filters.py

```diff
@@ -93,12 +93,11 @@
 
 ######################################################################
 #
-# The added noise is efficiently removed, as the image defaults are small (1
-# pixel wide), a small filter radius is sufficient. As the radius is
-# increasing, objects with bigger sizes are filtered as well, such as the
-# camera tripod. The median filter is often used for noise removal because
-# borders are preserved and e.g. salt and pepper noise typically does not
-# distort the gray-level.
+# The added noise is efficiently removed, as the image defaults are small (1-\
+# pixel wide), a small filter radius is sufficient. As the radius
+# increases, objects with bigger sizes get filtered as well, such as the
+# camera tripod. The median filter is often used for noise removal because it
+# preserves borders.
```

Comment:
Maybe something like

Maybe something like:

For example, consider noise that is located on only a few pixels in the entire image, as is the case for salt and pepper noise (reference to wikipedia), the median filter will ignore the noisy pixels since they will appear as outliers, and thus not significantly change the median of a group of local pixels in contrast to a moving average filter.

Reply:
I see. Thanks a lot, @hmaarrfk, I'll include your version.

Reply:
@hmaarrfk check out 67d2aaac340709f7 ;-)